### PR TITLE
Wrote a python script Using Gspread to automatically get data from Google Sheet.

### PR DESCRIPTION
**Pull request by olaemy**
@elseagle
@AmazingGrace-D
@tobaadesugba

### CODE REVIEW

File: README.md
Comment (elseagle) [inline README.md:1]: We have some instructions on discord that could help make your readme better, kindly check #task-submissions

File: gspread_new.ipynb
Comment (elseagle) [inline gspread_new.ipynb:1]: Kindly move this to a python script, we can't review a jupyter-notebook

**Review by elseagle — CHANGES_REQUESTED** [review]
Thanks for the PR, kindly checkout the comments

File: __pycache__/test_main.cpython-310-pytest-7.1.2.pyc
Comment (elseagle) [inline __pycache__/test_main.cpython-310-pytest-7.1.2.pyc:1]: remove all files like this from your commits. you don't need `*.pyc` files, add it to your gitignore

**Review by elseagle — APPROVED** [review]
Please checkout the requested changes before you merge

**Comment by olaemy** [comment]
Done, and commited all changes requested

**Pull request closed without merging**